src/libNode/FinalBlockProcessing.cpp

```diff
@@ -995,6 +1004,15 @@ bool Node::ProcessFinalBlockCore(uint64_t& dsBlockNumber,
       LOG_GENERAL(WARNING, "StoreFinalBlock failed!");
       return false;
     }
+    // Clear STL memory cache
+    if (!LOOKUP_NODE_MODE) {
+      auto clearStlMemoryCache = []() -> void {
+        LOG_GENERAL(INFO, "Clearing STL container cache for mining nodes");
```

Comment:
May be we have generic log like - releasing unused memory back to system

Reply:
I will remove this log

Reply:
Also, I will remove the time diff code logic.